### DISCUSSION TITLE
How to remove the RustDesk testing certifcate in Windows

### DISCUSSION THREAD

**Discussion by JRF63** [rustdesk/rustdesk#6268] [General]
~Not really easy to find it in the certificate manager.~

It's [WDKTestCert admin,133225435702113567](https://github.com/rustdesk/rustdesk/blob/48dbc06b294340347d799fa2ab321e9d162649de/src/platform/windows.rs#L1868) installed as ROOT. The provided "Uninstall RustDesk" removes it properly.

**Comment by b1naryzer0** [comment]
The uninstaller removes the certificate properly but leaves the store / folder entry untouched and doesn't remove it. As the folder name is different from installation to installation, this leaves one unneeded folder for each installation in the certificate store which can't be removed, neither via MMC nor powershell.
